impl/src/planning.rs

```diff
@@ -624,12 +652,21 @@ impl<'planner> CrateSubplanner<'planner> {
           )
         }
       }
+
+      // Check if the dependency has been renamed
+      dep.rename.as_ref().map(|rename| {
```

Comment:
Talked with @smklein about this a little later, so sorry for asking to change this again, but rust also has a [`if let`](https://doc.rust-lang.org/stable/rust-by-example/flow_control/if_let.html) pattern matching syntax.  That would probably be the clearest here.

```
if let Some(alias) = dep.rename.as_ref() {
  // everything in the closure
}
```

Reply:
Sounds good. Changed to `if let`